### PR TITLE
V1.10.0

### PR DESCRIPTION
**Pull request by david-saeger**
Updates lossOfSmell language and score

### CODE REVIEW

File: src/v1.10.0/index.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export * from './ages';
+export * from './conditions';
+export * from './exposures';
+export * from './symptoms';
+export * from './preExistingConditions';
+
+export * from './scale';
+export * from './types';
+export const calculatorVersion = '1.9.1';
```
Comment (SCasarottoDDS) [inline src/v1.10.0/index.ts:9]: 1.10.0?

**Review by SCasarottoDDS — CHANGES_REQUESTED** [review]
I think you also missed increasing the version in package.json (probably 1.3.0) and also updating the top level index.ts (src/index.ts) where you want to export it named as well as updating latest and updating the package version.

File: src/v1.10.0/index.ts (hunk 1 above)
Comment (david-saeger) [reply src/v1.10.0/index.ts]: yup, missed that

**Review by SCasarottoDDS — APPROVED** [review]
I think this should be good.

**Pull request merged**